### ISSUE TITLE
`hatch shell` throws error

### ISSUE THREAD

**Issue by carlosperate** [wheel-next/wheel-stub#11]
```bash
$ hatch shell
ERROR: In --require-hashes mode, all requirements must have their versions pinned with ==. These do not:
    setuptools from https://files.pythonhosted.org/packages/55/21/47d163f615df1d30c094f6c8bbb353619274edccf0327b185cc2493c2c33/setuptools-75.6.0-py3-none-any.whl (from nodeenv==1.8.0->-r /Users/microbit-carlos/workspace/tmp/rm-wheel/wheel-stub/requirements/requirements-lock.txt (line 49))
```

There is a reference to this in the `requirements/requirements-lock.txt` file, but it's unclear how to resolve it or procced, specially during the first steps trying the README instructions:

https://github.com/wheel-next/wheel-stub/blob/3f7a41d15086e1d11f43b4c1d245987993bfefa2/requirements/requirements-lock.txt#L158-L161

How would the `--allow-unsafe` be used with hatch?
I've never used hatch before, so maybe working around this is common knowledge for users of this tool, but I am not really sure of what to do in this case other than comment out the requirements file from pyproject.tom.

**Comment by emmatyping-nv** [comment]
Hi @carlosperate! Could you look at https://github.com/wheel-next/wheel-stub/pull/14 please? I think that should solve your issue.

**Comment by carlosperate** [comment]
Yes, I've tested it in a couple of Python environments and it works, thanks!
